packages/api/src/command/medical/document/document-query-storage-info.ts

```diff
@@ -20,54 +23,56 @@ export type S3Info = {
 };
 
 type SimpleFile = {
-  fhirDocId: string;
   docId: string;
   fileName: string;
   fileLocation: string;
 };
 
-export function docToFile(patient: Pick<Patient, "cxId" | "id">) {
+export function getDocToFileFunction(patient: Pick<Patient, "cxId" | "id">) {
   // TODO convert the input from CW Document to a Metriport shape
-  return (doc: Document): SimpleFile => {
-    const fhirDocId = getDocumentPrimaryId(doc);
-    const fileName = createS3FileName(patient.cxId, patient.id, fhirDocId);
-    return { fhirDocId, docId: doc.id, fileName, fileLocation: s3BucketName };
+  return async (doc: Document): Promise<SimpleFile> => {
+    const extension = getFileExtension(doc.content?.mimeType);
+    const docName = doc.id + (extension ? extension : "");
```

Comment:
nit: why not use a template literal as per the original code? saves a ternary op statement, and literals are faster in most cases

Reply:
The original code couldn't be used as-is, I had to code this from scratch. Ended up doing like this, but the literal would work too.

Both have ternary, and using a template string vs. a string concatenation def won't make a difference here.

But FWIW, I usually go and prefer template strings, not sure why ended up doing like this.

Reply:
updated